### PR TITLE
[DOC] Remove (newly unneeded) remarks about aliases

### PR DESCRIPTION
**Pull request by BurdetteLamar**
This covers *.* in the main directory.  I'll look into other offenses in lib/, ext/, etc.

### CODE REVIEW

File: re.c [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (jeremyevans) [inline re.c:1]: This looks accidental.

**Review by jeremyevans — CHANGES_REQUESTED** [review]
Looks good, only one inline comment.  OK to merge without further review after comment addressed.

File: re.c (hunk 1 above)
Comment (BurdetteLamar) [reply re.c]: Removed.

**Comment by BurdetteLamar** [comment]
Found one more in doc/string/length.rdoc, so requesting re-review.

**Review by jeremyevans — APPROVED** [review]

**Comment by BurdetteLamar** [comment]
I had a conflict that I resolved (I hope).  Pls check one more time.

**Review by jeremyevans — CHANGES_REQUESTED** [review]
This contains a merge commit, and Ruby does not support that.  Please rebase against the current master branch.  If you think that will be an issue, let me know and I'll handle it for you.

**Comment by BurdetteLamar** [comment]
> This contains a merge commit, and Ruby does not support that. Please rebase against the current master branch. If you think that will be an issue, let me know and I'll handle it for you.

I'm never comfortable with rebase.  Thanks for handling.

**Comment by jeremyevans** [comment]
Merged at 3b239d2

**Pull request closed without merging**